js/index.js

```diff
@@ -36,3 +36,77 @@ for (let i = 0; i < skills.length; i++) {
   // Append the list item element to the unordered list
   skillsList.appendChild(skill);
 }
+
+// Select the message form by name
+const messageForm = document.forms.leave_message;
```

Comment:
Since the lesson calls for DOM selection, I think they are looking for use of querySelector.
document.querySelector("[name='leave_message']");
I think the way you did is is clearer and simpler though, so no need to change.

Reply:
fixed